libs/common/src/lib/inject-map-api.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+import { SettingsService } from '@placeos/common';
+import { BehaviorSubject, Observable, Subject } from 'rxjs';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class InjectMapApiService {
+    private maps_key_exists = new BehaviorSubject<boolean>(null);
+
+    private _setKeyExists(value: boolean) {
+        this.maps_key_exists.next(value);
+    }
+
+    public getKeyExistsAsObservable(): Observable<boolean> {
```

Comment:
This can just be a readonly variable.
`public readonly key_exists$ = this.maps_key_exists.asObservable();`
Might be better to call it something like `is_initialised` or something as that better indicates what it represents.

Reply:
I've now updated this.